doc/manual/src/glossary.md

```diff
@@ -25,7 +32,8 @@
 
   Translate a [derivation] into a [store derivation].
 
-  See [`nix-instantiate`](./command-ref/nix-instantiate.md).
+  See [`nix-instantiate`](./command-ref/nix-instantiate.md), which translates
```

Comment:
I can't make a multi-line suggestion, but I would rephrase this as something like "...which produces store derivations from Nix expressions containing derivations". It's not crystal clear what `translate` means in this context.

Reply:
That sounds fine to me